community/cloud-pak.md

```diff
@@ -0,0 +1,160 @@
+
+Similar to the Helm CLI linter, IBM has created a linter specifically for IBM Cloud Pak certification.  There are three levels of messages produced by the linter:  
+
+`Information`: Style or cosmetic recommendations; not required for certification.
+
+`Warning`: These will not prevent successful deployment but may yield inconsistencies on the platform.  Strongly recommended to address but not strictly required for certification.
+
+`Error`: Error level checks are must-fix for certification.
+
+The linter is run by IBM against the chart in the pull request and then output is provided as a comment.
+
+The rules and descriptions for the content linter can be found here:
+[Lint rules for IBM Cloud Pak](lint-rules.md)
+
+## values-metadata.yaml
+
+IBM Cloud Private supports defining metadata for fields containing immutable or hidden fields, rendering booleans as check boxes, specifying allowed values, etc. to provide a rich deployment experience in the IBM Cloud Private GUI. This metadata is defined within a chart by using a file named `values-metadata.yaml`.  You must provide metadata in values-metadata.yaml for each parameter in values.yaml.  Detailed information of the format and syntax can be found in the IBM Cloud Private [Knowledge Center](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.1/app_center/values_metadata.html) For an example of how to use this file, refer to the [sample chart in this repository](https://github.com/IBM/charts/blob/master/community/sample-chart/values-metadata.yaml) or the many [IBM-provided charts in the repository](https://github.com/IBM/charts/tree/master/stable)
+
+## `ibm_cloud_pak` directory
+
+Your helm chart must include a new subdirectory `ibm_cloud_pak` which contains additional files specific to IBM Cloud Paks.  The `manifest.yaml` describes the Cloud Pak contents (charts and images) while the `qualification.yaml` specifies the level and duration of the Cloud Pak status.
+
+## ibm_cloud_pak/manifest.yaml
+
+IBM Cloud Private clusters are typically in an air-gapped environment with no access to the public internet.  When installing charts that point to a public image registry like Docker Hub, the chart pods will fail to pull the images.  To help alleviate this, IBM Cloud Private provides a local image registry as well as a local chart repository as well as tooling for building and installing binary packages in an air-gapped environment.
+
+IBM Cloud Private provides tooling to [build offline binary packages](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.1/manage_cluster/cli_catalog_commands.html#create-archive).  The tool reads a YAML manifest file in the helm chart and creates the binary package from the specified contents.  Cloud Paks must provide a such a manifest.yaml file for users to easily create offline binary packages in a consistent manner across products.  The manifest format can be found here: [IBM Cloud Pak manifest format](../reference/manifest-format.md)
```

Comment:
Update the link to:  `../spec/packaging/ibm_cloud_pak/manifest-yaml.md`

Reply:
Agreed